bluechi.spec.in

```diff
@@ -176,6 +184,10 @@ semanage port -a -t bluechi_port_t -p tcp 842 2>/dev/null || semanage port -m -t
 Summary:  BlueChi service controller command line tool
 Requires: %{name} = %{version}-%{release}
 
+%if 0%{?with_coverage}
+Requires: bluechi-coverage = %{version}-%{release}
```

Comment:
Why would everyone in the world want bluechi-coverage installed? At most this should be a suggests.